src/export/digitalPlanning/model.ts

```diff
@@ -323,7 +313,9 @@ export class DigitalPlanning {
 
   private getBoundary(): Payload["data"]["property"]["boundary"] {
     return {
-      site: this.passport.data?.["property.boundary.site"],
+      site: this.passport.data?.[
+        "property.boundary.site"
+      ] as unknown as GeoJSON,
```

Comment:
`as unknown as GeoJSON` is an especially odd casting but it seems to be what TS wanted! Am I missing something obvious?

Reply:
Yep I also had to do this elsewhere - it's the best we can do for now and is correct! 👌

Reply:
(+ it's whatever value we put in here is going to get parsed and validated shortly anyway)